### PR TITLE
Add new routing exercise and readme (Connects to #91)

### PR DESCRIPTION
**Pull request by moarsel**
Created simplified version of old routing exercise. This version has no boilerplate code included, except for in small snippets in the README.md. Any feedback on the difficulty and length would be helpful as well. 

(I can add a GIF demonstration if there aren't any major changes requested).

### CODE REVIEW

File: exercises/routing/README.md [hunk 1]
```diff
@@ -0,0 +1,164 @@
+
+## Step 1
+
+Using the `angular-cli` create a new project called `routing-exercise` with the `--routing` flag:
```
Comment (barretodavid) [inline exercises/routing/README.md]: ... in the `training` folder:

File: exercises/routing/README.md [hunk 2]
```diff
@@ -0,0 +1,164 @@
...
+ng generate service users
+```
+
+> The components and service to our ng-module automatically, so they are ready to use.
+
+## Step 3
+
+Add the following snippet to `app.component.css`:
+
+```
```
Comment (barretodavid) [inline exercises/routing/README.md]: You can add after the triple backtick the string `css` to have syntax highlight

File: exercises/routing/README.md [hunk 3]
```diff
@@ -0,0 +1,164 @@
...
+    id: "vn0ekj",
+    name: 'Jane Simmons',
+    company: 'Geoform',
+    email: 'Jane@geoform.me'
+  }, {
+    id: "f02jfi",
+    name: 'John Doe',
+    company: 'Flotonic',
+    email: 'john@flotonic.com'
+  }]
```
Comment (barretodavid) [inline exercises/routing/README.md]: I think it's better to move this data to its own JSON file called `users.json`. We can provide that file as part of the exercise like we did in the [structural directive exercise](https://github.com/rangle/angular-training-slides/tree/master/exercises/structural-directives)

File: exercises/routing/README.md [hunk 4]
```diff
@@ -0,0 +1,164 @@
...
+li a:hover {
+  background-color: #111;
+}
+```
+
+## Step 4
+
+Add the following snippet to the `app.component.html`
+
+```
```
Comment (barretodavid) [inline exercises/routing/README.md]: Same here, you can add the string `html` after the backtick to have syntax highlight

File: exercises/routing/README.md [hunk 5]
```diff
@@ -0,0 +1,164 @@
...
+
+## Step 6
+
+Import the `HomeComponent` and `UserListComponent`in `app-routing.module.ts`. Update the routes so that the default route loads the `HomeComponent` and the `"/users"` route loads the `UserListComponent`.
+
+## Step 7
+
+Update `users.service.ts` with the following snippet:
+
+```
```
Comment (barretodavid) [inline exercises/routing/README.md]: add the `ts` string after the backtick

File: exercises/routing/README.md [hunk 6]
```diff
@@ -0,0 +1,149 @@
+
```
Comment (barretodavid) [inline exercises/routing/README.md]: At the very beginning of the file we need to provide the overall goal of the exercise and show the gif. Take a look at this [example exercise](https://github.com/rangle/angular-training-slides/tree/master/exercises/structural-directives)

Comment (moarsel) [inline exercises/routing/README.md]: Thanks - I updated this along with some other wording improvements.

**Pull request merged**